threedi_modelchecker/checks/raster.py

```diff
@@ -210,10 +191,34 @@ def description(self):
         return f"The raster in {self.column_name} has non-square raster cells."
 
 
+class RasterGridSizeCheck(BaseRasterCheck):
+    """Check whether the global settings' grid size is an even multiple of a raster's cell size (at least 2x)."""
+
+    def get_invalid(self, session):
+        grid_space_query = session.query(models.GlobalSetting.grid_space).first()
+        if grid_space_query is not None:
+            self.grid_space = grid_space_query[0]
+        else:
+            return []
+        return super().get_invalid(session)
+
+    def is_valid(self, path: str, interface_cls: Type[RasterInterface]):
+        with interface_cls(path) as raster:
+            if not raster.is_valid_geotiff:
+                return True
+            # the x pixel size is used here,but it is equal to the y pixel size
+            return ((self.grid_space / raster.pixel_size[0]) % 2 == 0) and (
+                self.grid_space >= (2 * raster.pixel_size[0])
+            )
+
+    def description(self):
+        return f"{self.column_name} is not a positive even multiple of the raster cell size."
```

Comment:
I think that the column name here is `dem_file`, which is incorrect, please hardcode it like this:

```suggestion
        return f"v2_global_settings.grid_space is not a positive even multiple of the dem_file cell size."
```